src/commands/Quote.ts

```diff
@@ -0,0 +1,207 @@
+                    }
+                  }
+                }
+              }catch (error) {
+                console.log("Generator Type Find Is Not Working")
+              }
+            }
+            else if(this.generationType == "random" && Object.keys(userArray[value]).length > 0 ){
+              try{
+                var i = Math.floor(Math.random() * userArray[value].length);
+                this.discordMessage?.channel.send(i)
+                this.discordMessage?.channel.send(userArray[value][i].quote)
+                      //Will implement once more quotes are added.
+                      // else if(this.findType == "Random"){
+                      //   var a = Math.floor(Math.random() * userArray.length);
+                      //   var b = Math.floor(Math.random() * userArray[message.author.username].length);
+                      //   this.discordMessage?.channel.send(userArray[a].username)
+                      //   this.discordMessage?.channel.send(userArray[a][b].quote)
+                      // }
+              } catch(error){
+                console.log("Generator Type Random Is Not Working")
+              }
+            }
+            else if(this.generationType == "delete" && Object.keys(userArray[value]).length > 0 ){
+              try{
+                let number = this.findType[1]
+                if(this.findType[2] != null){
+                  number = number + this.findType[2]
+                }
+                if(this.findType[3] != null){
+                  number = number + this.findType[3]
+                }
+                if(parseInt(number)-1 <= Object.keys(userArray).length){
+                  userArray[value].splice(parseInt(number)-1,1)
+                  const data2 = fs.readFileSync('quotes2.json', 'utf8');
+                  const newUserArray = JSON.parse(data2)
+                  for(let i = 0; i < Object.keys(userArray).length; i = i + 1){
+                    for(let j = 0; j < Object.keys(userArray[i]).length; j = j + 1){
+                      try {
+                        if (i == 0) {
+                          newUserArray[0].push(userArray[i][j])
+                        } else if (i == 1) {
+                          newUserArray[1].push(userArray[i][j])
+                        } else if (i == 2) {
+                          newUserArray[2].push(userArray[i][j])
+                        } else if (i == 3) {
+                          newUserArray[3].push(userArray[i][j])
+                        } else if (i == 4) {
+                          newUserArray[4].push(userArray[i][j])
+                        }
+                      } catch (error) {
+                        console.log("JSON data cannot be saved")
+                        console.error(err);
+                      }
+                    }
+                  }
+                  fs.writeFileSync('quotes.json', JSON.stringify(newUserArray));
```

Comment:
Should not be deleting like this (using two files). Read in one file to `obj`, delete the correct quote, then write `obj` to the same file. No double loops required.